### PR TITLE
:sparkles: add plotly-resampler

### PR DESCRIPTION
**Pull request by jvdd**
For adding tutorials, packages, or apps
===========================================

##### What did you add to the README.md file?

* plotly-resampler

Have you ensured your contribution adheres to the pull request guidelines of [awesome](https://github.com/sindresorhus/awesome/edit/master/pull_request_template.md)? In particular:

- Not a duplicate.
- Only has awesome items. Awesome lists are curations of the best, not everything.
- Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Drop all the `A` / `An` prefixes in the descriptions.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.

----------

##### Do you want to be added to the contributors list on the README.md file?  

- [ ] Yes
- [x] No

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ A curated list of awesome Dash (plotly) resources
 - [dash-bootstrap-components](https://dash-bootstrap-components.opensource.asidatascience.com/) - Layout engine, default styles and high-level components based on Bootstrap.
 - [dash-uploader](https://github.com/np-8/dash-uploader) - Upload component for Dash. Supports large data files.
 - [Dash Mantine Components](https://github.com/snehilvj/dash-mantine-components) - Collection of 40+ Dash components based on Mantine React Components library.
+- [plotly-resampler](https://github.com/predict-idlab/plotly-resampler) - Wrapper for plotly figures that adds data aggregation functionality (via a dash app), enabling to visualize large sequences.
```
Comment (ucg8j) [inline README.md]:
```suggestion
- [plotly-resampler](https://github.com/predict-idlab/plotly-resampler) - Wrapper for plotly figures that adds data downsampling (aggregating) functionality, enabling the visualization of large datasets.
```

**Review by ucg8j — CHANGES_REQUESTED** [review]
Sorry for the late review 🙈 .

Thanks for the contribution. A small suggested change.

File: README.md (hunk 1 above)
Comment (jvdd) [reply README.md]: I agree with your suggestion!

**Comment by jvdd** [comment]
@ucg8j committed your suggestion! :smile:

**Pull request merged**